components/TextFields/tests/unit/Theming/TextFieldsMaterialThemingTests.swift

```diff
@@ -29,6 +29,11 @@ class TextFieldsMaterialThemingTests: XCTestCase {
   let filledIndicatorLineAlpha: CGFloat = 0.42
   let filledIconAlpha: CGFloat = 0.54
 
+  let outlinedTextFieldActiveAlpha: CGFloat = 0.87
```

Comment:
Do you need the `CGFloat` won't this work?
```
let outlinedTextFieldActiveAlpha = 0.87
```

Reply:
I was just following what was in the file before the change. FWIW, I think explicit types make Swift compilation faster... Probably not by much, but that makes sense, right?